File: OOPHP/Persoon.py
class Persoon:
  def __init__(self, naam, leeftijd, geslacht, is_student, gemiddeld_cijfer):
    self.naam = naam
    self.leeftijd = leeftijd
    self.geslacht = geslacht
    self.is_student = is_student
    self.gemiddeld_cijfer = gemiddeld_cijfer

  def __str__(self):
    return f"Naam: {self.naam}\nLeeftijd: {self.leeftijd}\nGeslacht: {self.geslacht}\nStudent: {'Ja' if self.is_student else 'Nee'}\nGemiddeld cijfer: {self.gemiddeld_cijfer}"

File: OOPHP/Persoon.php
<?php 
class Persoon {
    private $naam;
    private $leeftijd;
    private $geslacht;
    private $isStudent;
    private $gemiddeldCijfer;
  
    public function __construct($naam, $leeftijd, $geslacht, $isStudent, $gemiddeldCijfer) {
      $this->naam = $naam;
      $this->leeftijd = $leeftijd;
      $this->geslacht = $geslacht;
      $this->isStudent = $isStudent;
      $this->gemiddeldCijfer = $gemiddeldCijfer;
    }
}